admin/services/check-form.service.js

```diff
@@ -225,6 +225,92 @@ const checkFormService = {
       return line.trim()
     }).filter(Boolean)
     return result.length === config.LINES_PER_CHECK_FORM
+  },
+
+  /**
+   * Get unassigned forms for selected check window.
+   * @param checkWindowAssignedForms
+   * @returns {Promise<*>}
+   */
+  getUnassignedFormsForCheckWindow: async (checkWindowAssignedForms) => {
+    let checkFormData
+    let checkFormList = []
+
+    if (!checkWindowAssignedForms) {
+      return false
```

Comment:
Would it be better to return an empty array here?

Reply:
OK, but this requires modifying the corresponding view too.